features/basic_replication.py

```diff
@@ -0,0 +1,51 @@
+import psycopg2 as pg
+from time import sleep
+
+from lettuce import world, steps
+
+PATRONI_CONFIG = '{}.yml'
+
+
+@steps
+class BasicReplicationSteps(object):
+
+    def __init__(self, environ):
+        self.env = environ
+
+    def start_patroni(self, step, pg_name):
+        '''I start (\w+)'''
+        return world.pctl.start_patroni(pg_name)
+
+    def stop_patroni(self, step, pg_name):
+        '''I shut down (\w+)'''
+        return world.pctl.stop_patroni(pg_name)
+
+    def kill_patroni(self, step, pg_name):
+        '''I kill (\w+)'''
+        return world.pctl.stop_patroni(pg_name, kill=True)
+
+    def add_table(self, step, table_name, pg_name):
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Method could be a function](https://www.codacy.com/app/cyberdemn/patroni/file/2110318101/issues/source?bid=3127594&fileBranchId=3144623#l27)